client/src/stores/history/historyItemsStore.js

```diff
@@ -82,8 +86,10 @@ export const useHistoryItemsStore = defineStore("historyItemsStore", {
                 payload.forEach((item) => {
                     if (item.state == "ok") {
                         const itemCreateTime = new Date(item.create_time);
-                        if (itemCreateTime > state.latestCreateTime) {
+                        if (!state.latestCreateTime || itemCreateTime > state.latestCreateTime) {
                             state.latestCreateTime = itemCreateTime;
+                        } else if (itemCreateTime > state.lastUpdateTime) {
+                            state.lastUpdateTime = itemCreateTime;
```

Comment:
Thanks for fixing this feature, it works well. However, I am wondering if we can consolidate `lastUpdateTime` and `latestCreateTime` here.

Reply:
Yes, I wanted to ask you if I can completely remove `latestCreateTime` from the store even?

- `latestCreateTime` is the latest `item.create_time` for the current history (set in the `historyItemsStore`)
- `lastUpdateTime` is the `history.update_time` for the current history (set in `watchHistory`)

The only difference therefore, is that `latestCreateTime` does not take into account an item being hidden or deleted in the history, whereas, `lastUpdateTime` takes any change to the history in consideration

So, for now, we're only using `lastUpdateTime`. Hence, we could remove `latestCreateTime` unless we need it for some other reason in the future?

Reply:
That sounds good to me, consolidating it to `lastUpdateTime` should work fine. Thank you!